src/coreclr/jit/importer.cpp

```diff
@@ -807,8 +807,10 @@ GenTree* Compiler::impStoreStruct(GenTree*         store,
             WellKnownArg wellKnownArgType =
                 srcCall->ShouldHaveRetBufArg() ? WellKnownArg::RetBuffer : WellKnownArg::None;
 
-            GenTree*   destAddr = impGetStructAddr(store, CHECK_SPILL_ALL, /* willDeref */ true);
-            NewCallArg newArg   = NewCallArg::Primitive(destAddr).WellKnown(wellKnownArgType);
+            // TODO-Bug?: verify if flags matter here
```

Comment:
Can you explain this TODO please?

Reply:
This will now return flags like volatile or unaligned if the previous load contained them, as such I've marked all places that use this API where I wasn't sure if dropping them was okay.